webdriver/tests/support/fixtures_bidi.py

```diff
@@ -207,8 +208,8 @@ async def assert_pdf_dimensions(pdf, expected_dimensions):
         png = await render_pdf_to_png_bidi(pdf)
         width, height = png_dimensions(png)
 
-        assert cm_to_px(expected_dimensions["height"]) == height
-        assert cm_to_px(expected_dimensions["width"]) == width
+        assert height in [floor(cm_to_px(expected_dimensions["height"])), round(cm_to_px(expected_dimensions["height"]))]
```

Comment:
Note that we shouldn't use `round()` given that this has interesting side-effects for Python, which is rounding to the even side like `print(round(10.5))` is 10 and not 11.

I think that you as best should remove the last commit.

Reply:
OK, reverted to the initial approach.